src/testcase/statements/primitive/StringStatement.ts

```diff
@@ -75,7 +75,7 @@ export class StringStatement extends PrimitiveStatement<string> {
 
   addMutation(): StringStatement {
     const position = prng.nextInt(0, this.value.length - 1);
-    const addedChar = prng.pickOne(this.alphabet);
+    const addedChar = prng.pickOne(this.alphabet.split(""));
```

Comment:
Was this a bug?

Reply:
No, in javascript string and array are very similar so the original 'pickOne' function could pick a character from a string. But since in the new version of @syntest/core the types are stricter only arrays are allowed. 
Hence I convert the alphabet string to an array here.